src/README.md

```diff
@@ -0,0 +1,15 @@
+###### Adding Readme file
+Explaing how to use this script.
+### How to use this script
+This script is designed to be used with the following command line arguments:
+- `--input_file`: The input files should to be processed.
+- `--output_file`: The output file should  to be created.
+- `--num_lines`: The number of lines to be processed.
+- `--num_columns`: The number of columns must  be processed.
+- `--num_rows`: The number of rows must be processed.
+- `--num_files`: The number of files to be processed.
+
```

Comment:
Can you add more details about the files.

Reply:
changed as suggest